tools/junit-merger/junit-merger.go

```diff
@@ -62,6 +62,7 @@ func loadJUnitFiles(fileGlobs []string) (suites []reporters.JUnitTestSuite, err
 			if err != nil {
 				return nil, fmt.Errorf("failed to open file %s: %v", file, err)
 			}
+			defer f.Close()
```

Comment:
if you extract lines 61 - 68 to a function please,
it will be even faster, it will close the file when the new specific function returns, instead when 
loadJUnitFiles is finished handling all the files